build/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -39,17 +45,18 @@ export default () => {
       const behaviorName = path.basename(dir)
 
       let description
+      let specificationText
       const fileContent = fs.readFileSync(file.path).toString()
       const blockComments = extract(fileContent).filter(comment => comment.type === 'BlockComment') // filtering only block comments
-      const emptyDescriptionText = 'Behavior file has no description.'
 
-      // getting object that describes '@description' part of the comment's text
+      // getting object that describes '@description' and '@specification' part of the comment's text
       if (!_.isEmpty(blockComments)) {
         const commentTokens = doctrine.parse(blockComments[0].raw, { unwrap: true }).tags
-        const descriptionToken = commentTokens.find(token => token.title === 'description')
-        description = descriptionToken ? descriptionToken.description : emptyDescriptionText
-      } else {
-        description = emptyDescriptionText
+        const descriptionTokenDesc = getTextFromCommentToken(commentTokens, 'description')
+        const specificationTokenDesc = getTextFromCommentToken(commentTokens, 'specification')
+
+        specificationText = specificationTokenDesc ? specificationTokenDesc : ''
```

Comment:
no need to do that check, as it is already done in ```getTextFromCommentToken```, so it will return either value or empty string